src/button/AnchorButton.jsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import PropTypes from "prop-types";
+import ButtonStyles from "./button.style";
+
+export default function AnchorButton({
+  onClick,
+  type,
+  href,
+  target,
+  rel,
+  children,
+  customStyle
```

Comment:
Do we need custom style? You can always just add your own styles via css/styled components. Worst case you can always add !important

Reply:
I added for cases like where u want to override the width a default button has, it might be also that you would like to change the font-size or font-weight, stuff like that.